confidence/utils.py

```diff
@@ -51,6 +51,10 @@ def _split_keys(mapping, separator='.', colliding=None):
     Recursively walks *mapping* to split keys that contain the separator into
     nested mappings.
 
+    .. note::
+
+        All keys are coerced to `str` for use within a `.Configuration` object.
```

Comment:
Is this still correct?

Reply:
Good point, it is not. Fixed in e635d82.